content/media/index.md

```diff
@@ -45,6 +45,7 @@ A chronological archive of IPFS news, articles, media coverage, presence in conf
 - [20 January: JavaScript Apps go Inter-Planetary at Node+JS Interactive](https://medium.com/@italypaleale/watch-javascript-apps-going-inter-planetary-e33dba615a48)
 
 ### 2019
+- [19 December: A Technical Introduction to IPFS with Héctor Sanjuán - Our Networks 2019](https://www.youtube.com/watch?v=hnigvVuoaIA)
```

Comment:
The date was more like September I think

Reply:
September 22, 2019

Reply:
These have generally been listed by publication date of the artifact itself. Else things wind up buried in the list when something is released to public much later than original event.

Reply:
These have generally been listed by publication date of the artifact itself. Else things wind up buried in the list when something is released to public much later than original event.

Reply:
sgtm :+1: